packages/itwinui-react/src/core/Tree/TreeNode.tsx

```diff
@@ -83,6 +91,10 @@ type TreeNodeProps = {
    * Custom expander element. If `hasSubNodes` is false, it won't be shown.
    */
   expander?: React.ReactNode;
+  /**
+   * Props for content
+   */
+  contentProps?: React.CommonProps<any>;
```

Comment:
Think that this should also be `React.ComponentProps<any>`.

Reply:
yes that's a good catch